.changeset/rude-feet-end.md

```diff
@@ -0,0 +1,19 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - Hidden
+---
+
+**Hidden:** Add `component` support
+
+Provide a way to customise the DOM element rendered when using `Hidden`. If no `component` is specified it will fallback to the current behaviour — a `span` when setting `inline` to `true`, or otherwise a `div`.
```

Comment:
```suggestion
You can now customise the DOM element rendered when using `Hidden`. If no `component` is specified, it will fall back to the current behaviour — a `div` by default, or a `span` when setting `inline` to `true`.
```